Jenkinsfile

```diff
@@ -268,19 +266,32 @@ spec:
                                     def packageJSON = readJSON file: "package.json"
                                     String version = "${packageJSON.version}"
 
-                                    def notarizedZip = "${distFolder}/TheiaIDE-" + version + "-mac.zip"
+                                    def notarizedDmg = "${distFolder}/TheiaIDE.dmg"
+
+                                    // We'll mount and then copy the .app out of the DMG
+                                    def mountPoint = "${distFolder}/TheiaIDE-mount"
                                     def extractedFolder = "${distFolder}/TheiaIDE-extracted"
                                     def rezippedFile = "${distFolder}/TheiaIDE-rezipped.zip"
+                                    def finalZip = "${distFolder}/TheiaIDE-${version}-mac.zip"
+                                    sh "rm -rf \"${extractedFolder}\" \"${mountPoint}\""
+                                    sh "mkdir -p \"${extractedFolder}\" \"${mountPoint}\""
+                                    sh "hdiutil attach \"${notarizedDmg}\" -mountpoint \"${mountPoint}\""
+
+                                    // Copy the .app from the DMG to a folder we can zip
+                                    sh "ditto \"${mountPoint}/TheiaIDE.app\" \"${extractedFolder}/TheiaIDE.app\""
```

Comment:
A nit for the next iteration of this script: we can just create the archive via `ditto` straight out of the mounted DMG. No need to copy to the main filesystem, first.